packages/mobile/src/invite/saga.ts

```diff
@@ -412,7 +417,7 @@ export function* withdrawFundsFromTempAccount(
 
   const tx: CeloTransactionObject<boolean> = yield call(
     createTokenTransferTransaction,
-    CURRENCY_ENUM.DOLLAR,
+    currency ? currency : CURRENCY_ENUM.DOLLAR,
     {
       recipientAddress: newAccount,
       amount: netSendAmount,
```

Comment:
Should update `SENTINEL_INVITE_COMMENT` here and in fee estimation above to some migration comment